.gitignore

```diff
@@ -51,7 +51,6 @@ hgignore[.-]*
 ignore[.-]*
 svnignore[.-]*
 lint.db
-build
```

Comment:
We used to use `build` as a temp folder. This is now used by nuke as the build project.